docs/concepts/reftypes.mdx

```diff
@@ -0,0 +1,221 @@
+---
+title: Attached Artifacts
+sidebar_position: 1
+---
+
+# Attaching and listing reference types
+
+The OCI distribution specification allows us to "link" [OCI artifacts](artifact.mdx)
+through the use of the concept of reference types.
+Artifacts can be tied to one another by defining one as a subject of another.
+This is crucial for maintaining dependencies between different artifacts and
+enabling the movement of artifacts and their dependencies between environments.
+
+## Implementing Reference Types in OCI artifacts
+
+The concept of linking or relating OCI artifacts is achieved using the concept
+of reference types, where one artifact can be set as a subject to another.
+In this context, "subject" refers to the target of a reference.
+
+### The Art of Associating Artifacts
+
+Consider a typical cloud-native environment with countless OCI artifacts.
+Often, it's imperative to manage relationships between them—say, an OCI artifact's
+signatures, or metadata about an artifact. This is where reference types come into the picture.
+
+A reference type enables an OCI artifact to refer to another, creating a defined
+link or association between them. This association can be for various purposes,
+such as versioning, updates, signatures, or dependencies.
+
+![OCI artifact with refererrs pointing to the artifact](/img/concepts/reftypes.png)
+
+## Creating a Reference Artifact
+
+OCI Artifacts may be stored in a content-addressable storage system or OCI Layout directory.
+For the purpose of this guide, we'll use the OCI Distribution Specification to
+create and list reference types.
+
+The creation of a reference artifact is a two-step process:
+
+1. **Step One**: Put an artifact, say Artifact A, using the PUT Manifest and
+blob upload APIs if the artifact has blob content.
+    - The prerequisite for a reference type is that the subject artifact must exist in the repository.
```

Comment:
```suggestion
> [!NOTE]  
> The prerequisite for a reference type is that the subject artifact must exist in the repository.
```

Reply:
Actually now that I read it back, there is no requirement that the subject should exist :) I need to remove that. Thanks for the catch.

Reply:
Added a note to highlight that its not required at the bottom.